template/cordova/lib/log.js

```diff
@@ -0,0 +1,227 @@
+    }).then(function(state) {
+        adminLogInitialState = state;
+        return enableLogging('Microsoft-Windows-AppHost/Admin');
+    }).then(function () {
+        return enableLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+    }).then(function () {
+        console.log('Now printing logs. To stop, please press Ctrl+C once.');
+        startLogging('Microsoft-Windows-AppHost/Admin');
+        startLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+    }).catch(function (error) {
+        console.error(error);
+    });
+
+    // Disable logs before exiting
+    process.on('exit', function () {
+        if ((appTracingLogInitialState !== null) && !appTracingLogInitialState) {
+            disableLogging('Microsoft-Windows-AppHost/ApplicationTracing');
+        }
+        if ((adminLogInitialState !== null) && !adminLogInitialState) {
+            disableLogging('Microsoft-Windows-AppHost/Admin');
+        }
+    });
+
+    // Catch Ctrl+C event and exit gracefully
+    process.on('SIGINT', function () {
+        process.exit(2);
+    });
+
+    // Catch uncaught exceptions, print trace, then exit gracefully
+    process.on('uncaughtException', function(e) {
+        console.log(e.stack);
+        process.exit(99);
+    });
+};
+
+function startLogging(channel) {
+    var startTime = new Date().toISOString();
+    setInterval(function() {
+        var command = 'wevtutil qe ' + channel + ' /q:"*[System [(TimeCreated [@SystemTime>\'' + startTime + '\'])]]" /e:root';
+        cp.exec(command, function (error, stdout, stderr) {
+            if (error) {
+                throw new Error('Failed to run wevtutil command. ' + error);
+            } else {
+                parseEvents(stdout).forEach(function (evt) {
+                    startTime = evt.timeCreated;
+                    console.log(stringifyEvent(evt));
+                });
+            }
+        });
+    }, 1000);
+}
+
+module.exports.help = function() {
+    console.log('Usage: ' + path.relative(process.cwd(), path.join(ROOT, 'cordova', 'log')));
+    console.log('Continuously prints the windows logs output to the command line.');
+    process.exit(0);
+};
+
+function getElementValue(et, element, attribute) {
+    var result;
+
+    var found = et.findall(element);
+    if (found.length > 0) {
+        if (!!attribute) {
+            result = found[0].get(attribute);
+        } else {
+            result = found[0].text; 
+        }
+    }
+
+    return result;
+}
+
+function parseEvents(output) {
+    var etree = et.parse(output);
+    var events = etree.getroot().findall('./Event');
+    var results = [];
+
+    events.forEach(function (event) {
+        // Get only unhandled exceptions from Admin log
+        if (getElementValue(event, './System/Channel') === 'Microsoft-Windows-AppHost/Admin' &&
+            typeof getElementValue(event, './UserData/WWAUnhandledApplicationException') === 'undefined') {
```

Comment:
Here's the event that got generated:
+ System 

  - Provider 

   [ Name]  Microsoft-Windows-AppHost 
   [ Guid]  {98E0765D-8C42-44A3-A57B-760D7F93225A} 
 
   EventID 127 
 
   Version 0 
 
   Level 2 
 
   Task 8 
 
   Opcode 12 
 
   Keywords 0x8000000000000001 
 
  - TimeCreated 

   [ SystemTime]  2015-08-12T00:47:00.808688300Z 
 
   EventRecordID 100 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  10600 
   [ ThreadID]  14852 
 
   Channel Microsoft-Windows-AppHost/Admin 
 
   Computer nikhilkh005.redmond.corp.microsoft.com 
 
  - Security 

   [ UserID]  S-1-5-21-2127521184-1604012920-1887927527-2523208 
 

- UserData 

  - WWATerminateApplication 

   DisplayName HelloCordova 
 
   ApplicationName io.cordova.hellocordova 
 
   AppUserModelId io.cordova.hellocordova_h35559jr9hy9m!io.cordova.hellocordova 
 
   PackageFullName io.cordova.hellocordova_0.0.1.0_neutral__h35559jr9hy9m 
 
   ProcessId 10600 
 
   ErrorDescription {"description":"'platform' is undefined","number":-2146823279,"stack":"ReferenceError: 'platform' is undefined\n at activationHandler (ms-appx://io.cordova.hellocordova/www/cordova.js:1320:17)\n at dispatchOne (ms-appx://microsoft.winjs.2.0/js/base.js:9416:25)\n at dispatchEvent (ms-appx://microsoft.winjs.2.0/js/base.js:9415:21)\n at drainOneEvent (ms-appx://microsoft.winjs.2.0/js/base.js:9469:13)\n at drainQueue (ms-appx://microsoft.winjs.2.0/js/base.js:9486:9)\n at Anonymous function (ms-appx://microsoft.winjs.2.0/js/base.js:2952:17)\n at state_running.enter (ms-appx://microsoft.winjs.2.0/js/base.js:2951:13)\n at _setState (ms-appx://microsoft.winjs.2.0/js/base.js:2667:13)\n at Anonymous function (ms-appx://microsoft.winjs.2.0/js/base.js:2866:13)\n at _execute (ms-appx://microsoft.winjs.2.0/js/base.js:2642:13)"} 
 
   DocumentFile /www/index.html 
 
   StackTrace ReferenceError: 'platform' is undefined at activationHandler (ms-appx://io.cordova.hellocordova/www/cordova.js:1320:17) at dispatchOne (ms-appx://microsoft.winjs.2.0/js/base.js:9416:25) at dispatchEvent (ms-appx://microsoft.winjs.2.0/js/base.js:9415:21) at drainOneEvent (ms-appx://microsoft.winjs.2.0/js/base.js:9469:13) at drainQueue (ms-appx://microsoft.winjs.2.0/js/base.js:9486:9) at Anonymous function (ms-appx://microsoft.winjs.2.0/js/base.js:2952:17) at state_running.enter (ms-appx://microsoft.winjs.2.0/js/base.js:2951:13) at _setState (ms-appx://microsoft.winjs.2.0/js/base.js:2667:13) at Anonymous function (ms-appx://microsoft.winjs.2.0/js/base.js:2866:13) at _execute (ms-appx://microsoft.winjs.2.0/js/base.js:2642:13)